tests/integration/modifiers/did-resize-test.js

```diff
@@ -87,7 +81,6 @@ module('Integration | Modifier | did-resize', function (hooks) {
 
     await render(hbs`<div {{did-resize this.resizeStub}}></div>`);
 
-    assert.notOk(resizeCallback, 'no callback received');
```

Comment:
I had to remove this `assert` since the observer is now `static` and `resizeCallback` might be set by the other test cases. I don't think this would be an issue as the next `assert` is the important one imo.

Reply:
Wouldn't leaving `resizeCallback = null` in the `beforeEach` eliminate this issue by ensuring it gets reset before every test?

Reply:
If we do that, I think the issue is that it won't be set back to the callback because only the test case that runs first will create a new instance of the mock observer, after that it is shared by caching it using the static property.

Reply:
You're right! I was mixing up the Observer and the Modifier itself here.